src/declarations/stencil-private.ts

```diff
@@ -2124,6 +2124,7 @@ export interface Url {
 
 declare global {
   namespace jest {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Nit - similar to `// @ts-ignore` comments, can we add the reason as to why we have to disable a rule? That way folks reading the code in the future understand why the `// eslint-disable*` directive is there.  In ESLint, separating the reason from the directive itself is done with 2+ `-`'s:

```diff
    // eslint-disable-next-line @typescript-eslint/no-unused-vars -- the reason we're disabling this rule
```